### PR TITLE
Added msort

### PR DESCRIPTION
**Pull request by ZJay07**
#22245

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by jieunboy0516** [comment]
close #22245

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -1088,6 +1088,9 @@ def atan2_(self, other):
     def fmin(self, other):
         return torch_frontend.fmin(self, other)
 
+    def msort(self, out=None):
```
Comment (jieunboy0516) [inline ivy/functional/frontends/torch/tensor.py:1091]: I think the out is not needed here

**Comment by jieunboy0516** [comment]
Hi, thanks for your contribution to Ivy! I have left a comment to remove the `out` , but everything else looks good to me!

File: ivy/functional/frontends/torch/tensor.py (hunk 1 above)
Comment (ZJay07) [reply ivy/functional/frontends/torch/tensor.py]: Removed in the latest commit, thanks for spotting this out (pun)!

**Pull request merged**